### PR TITLE
Updated Croatian translation

### PR DESCRIPTION
**Pull request by sheeepdev**
tldr:

- added everything that was added in recent updates
- renamed hr_HR to just hr for simplicity sake because hr_HR and normal hr are virtually the same

+ i dont know if i messed up but where there are supposed to be multiple `msgstr`'s i think i might have created non required ones, sorry poedit is kinda confusing

thanks

### CODE REVIEW

File: po/hr.po [hunk 1]
```diff
@@ -310,10 +308,9 @@ msgstr "Otvori datoteku"
 #. TRANSLATORS: Add your name/nickname here
 #: src/window.py:284 src/window.py:285
 msgid "translator-credits"
-msgstr ""
+msgstr "sheep"
 
 #: src/ui/albumcoverbutton.ui:73
-#, fuzzy
 msgid "Album cover"
 msgstr "Album"
```
Comment (knuxify) [inline po/hr.po:315]: You might've forgotten to translate "Album cover"

**Review by knuxify — CHANGES_REQUESTED** [review]

**Comment by knuxify** [comment]
> i dont know if i messed up but where there are supposed to be multiple msgstr's i think i might have created non required ones, sorry poedit is kinda confusing

if you mean the multiple msgstrs for the "n files selected" string, those are for plural forms; this depends on the language but for languages that can have multiple plural forms depending on the number, there are simply multiple fields (like for example in Polish where for 1 it's "Wybrano 1 plik", and for more than 1 it's either "Wybrano 2 pliki" (2, 3, 4...) or "Wybrano 5 plików" (5, 6, 7, 8, 9, 10...)). seems like this is also the case in Croatian; https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html is a handy reference for this, in your case https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#hr

**Comment by sheeepdev** [comment]
> > i dont know if i messed up but where there are supposed to be multiple msgstr's i think i might have created non required ones, sorry poedit is kinda confusing
> 
> if you mean the multiple msgstrs for the "n files selected" string, those are for plural forms; this depends on the language but for languages that can have multiple plural forms depending on the number, there are simply multiple fields (like for example in Polish where for 1 it's "Wybrano 1 plik", and for more than 1 it's either "Wybrano 2 pliki" (2, 3, 4...) or "Wybrano 5 plików" (5, 6, 7, 8, 9, 10...)). seems like this is also the case in Croatian; [unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html) is a handy reference for this, in your case [unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#hr](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#hr)

Yeah it just confused me for more then one because there were three fields for numbers. In Croatian plural is same for 2, 5, 6 or whatever number bigger than one. Should I just remove the 3rd `msgstr` then?

File: po/hr.po (hunk 1 above)
Comment (sheeepdev) [reply po/hr.po]: Fixed!

**Comment by knuxify** [comment]
> Should I just remove the 3rd msgstr then?

nope, it's fine as-is.

thanks for the contribution!

**Pull request merged**